cmd/monitor.go

```diff
@@ -19,51 +22,70 @@ func main() {
 	//  cli for stop/start
 	//  cli for service creation
 
-	jFile, err := ioutil.ReadFile(controler.CFG_PATH)
+	_, err := flags.Parse(&config)
 	if err != nil {
-		panic(err)
-	}
-	var cfg Conf
-	err = json.Unmarshal(jFile, &cfg)
-	if err != nil {
-		panic(err)
+		os.Exit(1)
 	}
 
-	monitor := controler.NewServiceController()
+	monitor := controler.NewServiceControllerByPath(config.ConfigFile)
+
+	var access controler.Access = monitor
 
 	router := gin.Default()
-	router.GET("/monitor/run/:name", func(gctx *gin.Context) {
+	authOnly := router.Group("/monitor").Use(func(gctx *gin.Context) {
+		const realm = "Authorization Required"
```

Comment:
maybe we should storage all consts in one place? :)

Reply:
Yes, we mostly should, however it is a routing parameters. Much easier to navigate through the application structure regarding of web mapping to the controller